kernel/src/asm.S

```diff
@@ -154,6 +154,11 @@ _xous_resume_context:
     move        sp, a0
 
     LOAD        x1, 0*REGBYTES(sp)
+    // Clear any reservations, invalidating any atomic operations
+#ifndef SKIP_MULTICORE
```

Comment:
what's the intent of the `#ifndef`? i don't see it defined anywhere else so the intention is to have this included then, yes?